airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/spec.py

```diff
@@ -65,6 +65,18 @@ class Config:
         description="profile Ids you want to fetch data for",
     )
 
+    report_wait_timeout: int = Field(
+        None,
+        name="Report Wait Timeout",
+        description="Timeout duration for Reports"
```

Comment:
@aakashkumarmaple can you add default values and examples here?

Reply:
Resolved. Added Default values and examples